test/metabase/db/schema_migrations_test.clj

```diff
@@ -491,22 +491,22 @@
                 id))
             (all-user-perms []
               (db/query {:select [:object], :from [Permissions], :where [:= :group_id (all-users-group-id)]}))]
-      (impl/test-migrations ["v43.00-020"] [migrate!]
+      (impl/test-migrations ["v43.00-020" "v43.00-021"] [migrate!]
```

Comment:
with the newly added migration, these tests will be affected, thus limiting its end migration.